src/pages/index.js

```diff
@@ -134,9 +126,8 @@ function handleProfileFormSubmit({ name, description }) {
       userInfo.setUserInfo(data.name, data.about);
       editProfilePopup.close();
     })
-    .catch((err) => {
-      console.error(err);
-    })
+    .catch(console.error)
+
     .finally(() => {
       editProfilePopup.setLoading(false, "Save");
```

Comment:
You don't have `setLoading` in your project. Now it's called `renderLoading`. That's why there are errors when I try to submit a form